pkg/controller/component/rsm_convertor.go

```diff
@@ -88,8 +88,14 @@ func (c *rsmMemberUpdateStrategyConvertor) convert(args ...any) (any, error) {
 type rsmPodManagementPolicyConvertor struct{}
 
 func (c *rsmPodManagementPolicyConvertor) convert(args ...any) (any, error) {
-	// componentDefinition does not define PodManagementPolicy and StatefulSetUpdateStrategy.
-	// The Parallel strategy is used by default here. If necessary, the componentDefinition API can be expanded later
+	synthesizedComp, err := parseRSMConvertorArgs(args...)
+	if err != nil {
+		return nil, err
+	}
+	memberUpdateStrategy := getMemberUpdateStrategy(synthesizedComp)
+	if memberUpdateStrategy == nil || *memberUpdateStrategy == workloads.SerialUpdateStrategy {
```

Comment:
'sequential' is better than 'serial'

Reply:
Serial is a strategy that has been defined and is being used.